src/plugins/charts/BarGraphView.vue

```diff
@@ -93,33 +80,9 @@ export default {
     methods: {
         addTelemetryObject(telemetryObject) {
             const key = this.openmct.objects.makeKeyString(telemetryObject.identifier);
```

Comment:
Could we add the name of the telemetry object to `configuration.seriesStyles` when addTelemetryObject happens here?